app/workers/scheduler/trending_tags_scheduler.rb

```diff
@@ -6,6 +6,6 @@ class Scheduler::TrendingTagsScheduler
   sidekiq_options retry: 0
 
   def perform
-    TrendingTags.update! if Setting.trends
+    Trends.refresh!
```

Comment:
Why run the worker if the feature is disabled?

Reply:
Because it’s useful information for the admin without being publicly visible and the entire hashtags view is now centered on that sorting

Reply:
ok I guess hashtags might be interesting, but I can't see why for links, as there is no way for a regular admin to see who posted links or the context

Reply:
Maybe there will be